blocks/library/search/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+ import './editor.scss';
```

Comment:
We no longer import SCSS files in JavaScript. There is `editor.scss` file where this import should be added instead:
https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/editor.scss